core/block_svg.js

```diff
@@ -975,7 +977,7 @@ Blockly.BlockSvg.prototype.applyColour = function() {
     return;
   }
 
-  this.pathObject.applyColour(this.isShadow());
+  this.pathObject.applyColour(this.isShadow(), this.getParent());
```

Comment:
How does this pass type checks?  Only some versions of `applyColour` take two arguments now.